### ISSUE TITLE
Asymmetric lateral ventricle

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7407]
**Preferred term label:**
Asymmetric lateral ventricle

**Synonyms**
ALV; Lateral ventricular asymmetry

**Definition (free text, please give PubMed ID)**
An abnormal difference between the left and right sides of the lateral ventricles (PMID: 18253688; 2257506).

**Comment**
The degree of ALV is classified as mild, moderate or severe, according to the ratio of the larger frontal horn diameter to the smaller one.

**Parent term (use hpo.jax.org/app)**
Asymmetric ventricles HP:0100960

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
This is a synonym of [Asymmetric ventricles HP:0100960 ](https://hpo.jax.org/app/browse/term/HP:0100960). I updated the label to Lateral ventricular asymmetry, which is clearer and added a definition and these references. Thanks!
